src/main/scala/beam/agentsim/agents/choice/mode/ModeChoiceLCCM.scala

```diff
@@ -86,18 +87,18 @@ class ModeChoiceLCCM(
           "numCars"  -> attributesOfIndividual.householdAttributes.numCars,
           "numBikes" -> attributesOfIndividual.householdAttributes.numBikes
         )
-        AlternativeAttributes("dummy", theParams)
+        Map("dummy" -> theParams)
       }
 
       val classMembershipInputData =
-        lccm.classMembershipModels.head._2.alternativeParams.keySet.map { theClassName =>
+        lccm.classMembershipModels.head._2.utilityFunctions.keySet.map { theClassName =>
           val modeChoiceExpectedMaxUtility = lccm
             .modeChoiceModels(tourType)(theClassName)
-            .getExpectedMaximumUtility(modeChoiceInputData)
+            .getExpectedMaximumUtility(modeChoiceInputData.toMap)
           val surplusAttrib: Map[String, Double] =
-            Map("surplus" -> modeChoiceExpectedMaxUtility)
-          AlternativeAttributes(theClassName, attribIndivData.attributes ++ surplusAttrib)
-        }.toVector
+            Map("surplus" -> modeChoiceExpectedMaxUtility.getOrElse(0))
+          (theClassName, attribIndivData.head._2 ++ surplusAttrib)
```

Comment:
I realize that this is hard-coded directly above and will currently never be empty, but we should add a note/test/something (empty check?) to make sure this is always the case.